File: siserp-logistica-frontend/src/main/siserp-logistica-frontend/src/app/modules/intranet/dto/request/producto-registrar.request.ts
export class ProductoRegistrarRequest {
    id: number;
    idMarca: number;
    nomMarca: string;
    idtTipo: number;
    nomTipo: string;
    idtUnidadMedida: number;
    nomUnidadMedida: string;
    codigo: string;
    nombre: string;
    descripcion: string;
    flagActivo: number;
    idUsuarioCrea: number;
    fecUsuarioCrea: Date;
    idUsuarioMod: number;
    fecUsuarioMod: Date;
}

File: siserp-logistica-backend/src/main/java/com/besoft/siserp/lgtc/dto/request/ProductoRegistrarRequest.java
package com.besoft.siserp.lgtc.dto.request;

import java.io.Serializable;
import java.util.Date;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class ProductoRegistrarRequest implements Serializable {

	private static final long serialVersionUID = 1L;

	private Long id;
	private Long idMarca;
	private String nomMarca;
	private Integer idtTipo;
	private String nomTipo;
	private Integer idtUnidadMedida;
	private String nomUnidadMedida;
	private String codigo;
	private String nombre;
	private String descripcion;
	private Integer flagActivo;
	private Long idUsuarioCrea;
	private Date fecUsuarioCrea;
	private Long idUsuarioMod;
	private Date fecUsuarioMod;

}